libs/cmdio/io.go

```diff
@@ -190,13 +184,25 @@ func (c *cmdIO) Select(names map[string]string, label string) (id string, err er
 	return
 }
 
+// Show a selection prompt where the user can pick one of the name/id items.
+// The items are sorted alphabetically by name.
 func Select[V any](ctx context.Context, names map[string]V, label string) (id string, err error) {
 	c := fromContext(ctx)
-	stringNames := map[string]string{}
+	var items []Tuple
 	for k, v := range names {
-		stringNames[k] = fmt.Sprint(v)
+		items = append(items, Tuple{k, fmt.Sprint(v)})
 	}
-	return c.Select(stringNames, label)
+	slices.SortFunc(items, func(a, b Tuple) int {
+		return strings.Compare(a.Name, b.Name)
+	})
+	return c.Select(items, label)
+}
+
+// Show a selection prompt where the user can pick one of the name/id items.
+// The items can appear in a custom order and are not alphabetically sorted.
```

Comment:
nit: What do you think about? Otherwise, it's not clear what the custom order is.
```suggestion
// The items appear in the order specified In the "items" argument and are not alphabetically sorted.
```